tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc

```diff
@@ -248,6 +248,10 @@ attribute is optional. Implementations can also add their own attributes.
 Other MicroProfile specifications can add their own attributes under their own attribute name following the
 convention `mp.[specification short name].[attribute name]`.
 
+Implementation libraries must set the library name and version using the following properties:
+`mp.telemetry.tracing.name`
```

Comment:
I was told these properties are not required in the latest release of Open Telemetry. We might not need it any more.

Reply:
or it can be changed from must to may.